cmake/GenerateBuiltins.cmake

```diff
@@ -291,6 +292,7 @@ endfunction()
 function(builtin_xe_to_cpp bit resultFileName)
     set(inputFilePath builtins/builtins-cm-${bit}.ll)
     set(SKIP OFF)
+    # TODO: Look into cross-compiling for windows/linux
```

Comment:
@nurmukhametov I wasn't too sure what kind of TODO you were after, something like this?

Reply:
Yes, it is fine.